language-support/java/codegen/src/it/daml/Tests/Escape.daml

```diff
@@ -36,6 +41,9 @@ template EscapeTemplates
     with
         int : Int
         owner : Party
+        r : EscapeRecords () () ()
+        v : EscapeVariants () ()
```

Comment:
These tests were previously not effective before as the types were not a part of a template and thus were filtered out from code-gen.